src/__tests__/__snapshots__/transpiled-code.ts.snap

```diff
@@ -0,0 +1,514 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Empty code returns empty string 1`] = `
+Object {
+  "code": "",
+  "transpiled": "",
+}
+`;
+
+exports[`Ensure return values are of the form {isTail...} after function declarations are transformed 1`] = `
+Object {
+  "code": "function f() {
+  return null;
+}",
+  "transpiled": "const native = $$NATIVE_STORAGE;
+const boolOrErr = native.operators.get(\\"itselfIfBooleanElseError\\");
+const callIfFuncAndRightArgs = native.operators.get(\\"callIfFunctionAndRightArgumentsElseError\\");
+const wrap = native.properTailCalls.wrap;
+(( <globals redacted> ) => {
+  return (() => {
```

Comment:
Now that I'm actually looking at the transpiled output, what's the purpose of this inner wrap?

Reply:
```js
const native = $$NATIVE_STORAGE;
const boolOrErr = native.operators.get(\\"itselfIfBooleanElseError\\");
const callIfFuncAndRightArgs = native.operators.get(\\"callIfFunctionAndRightArgumentsElseError\\");
const wrap = native.properTailCalls.wrap;
(( <globals redacted> ) => { // (A) list globals as parameters here to set to undefined at A2
  // if there's no inner wrap, 
  // const Number or const window would be illegal,
  // but it doesn't make sense to not allow them as variable names.
  return (() => {
    for (1; 1; 1) {}
    const lastStatementResult = eval(\\"1;\\");
    return lastStatementResult;
  })();
})(); // A2
```

Reply:
I see, thanks.